components/EventDetail.js

```diff
@@ -9,43 +14,173 @@ const StyledContainer = styled.div`
   max-width: 36rem;
   margin-inline: auto;
 `;
+
 const StyledHeaderImage = styled(Image)`
   width: 100%;
 `;
+
 const StyledTitle = styled.p``;
```

Comment:
Seems like you do not need this Styled Component as there are no styles defined. If you do not need it, please delete it 🧹 
```suggestion

```